src/components/center/center.test.tsx

```diff
@@ -1,15 +1,17 @@
 import React from 'react';
 import { render } from '@testing-library/react';
 
+import { testA11y } from '../../test-utils';
 import { Center } from './center';
 
-test('it renders with both vertical and horizontal by default', () => {
+test('it renders with both vertical and horizontal by default', async () => {
```

Comment:
Do we want to test for vertical  / horizontal cases separately too?

Reply:
there are tests for them individually, but if you mean test a11y its unnecessary as they just class applications.